### PR TITLE
574-topology-detail-show-teams

### LINKED ISSUE #574: Topology Detail - Show teams

### PR DESCRIPTION
**Pull request by mainawycliffe**
Closes #574

### CODE REVIEW

**Comment by mainawycliffe** [comment]
@moshloop is this what you had in mind

Link: https://deploy-preview-689--goofy-euclid-75956c.netlify.app/topology//topology/0184f511-0ec7-397f-b7eb-c1642b149889?sortBy=status&sortOrder=desc
image

File: src/components/TopologySidebar/ComponentTeams.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      Header={
+        <div className="flex py-2 flex-row flex-1 items-center">
+          <Title
+            title="Teams"
+            icon={<AiOutlineTeam className="w-6 h-auto" />}
+          />
+        </div>
+      }
+    >
+      <div className="flex flex-col space-y-4 py-2 max-w-full">
```
Comment (moshloop) [inline src/components/TopologySidebar/ComponentTeams.tsx:35]: Why do we need to apply styles on the header and body?

File: src/components/TopologySidebar/ComponentTeams.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+type Props = {
+  componentId: string;
+};
+
+export function ComponentTeams({ componentId }: Props) {
+  const { data: componentTeams, isLoading } = useGetComponentsTeamQuery(
+    componentId,
+    {
+      onError: (error) => {
```
Comment (moshloop) [inline src/components/TopologySidebar/ComponentTeams.tsx:18]: Shouldn't this be handled by the axios interceptor ?

File: src/components/TopologySidebar/ComponentTeams.tsx (hunk 1 above)
Comment (mainawycliffe) [reply src/components/TopologySidebar/ComponentTeams.tsx]: @moshloop removed the header styles, but body styles are needed to ensure vertical spacing (`space-y-4`) for the list of items.

File: src/components/TopologySidebar/index.tsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ import Configs from "../Sidebars/configs";
 import Incidents from "../Sidebars/incidents";
```
Comment (Prashanth-Nelli) [inline src/components/TopologySidebar/index.tsx:4]: nit: rename this file name to TopologySidebar.tsx

**Review by Prashanth-Nelli — APPROVED** [review]

**Pull request merged**